x-pack/plugins/security_solution/public/cases/components/all_cases/table_filters.tsx

```diff
@@ -41,7 +41,12 @@ const StatusFilterWrapper = styled(EuiFlexItem)`
  * @param onFilterChanged change listener to be notified on filter changes
  */
 
-const defaultInitial = { search: '', reporters: [], status: CaseStatuses.open, tags: [] };
+const defaultInitial = {
+  search: '',
+  reporters: [],
+  status: AllCaseType as CaseStatusFilter,
```

Comment:
nit: In combination with https://github.com/elastic/kibana/pull/91863/files#r583515280 it can be done as `status: AllCaseType`